drivers/mrf24j40/include/mrf24j40_registers.h

```diff
@@ -312,6 +312,30 @@ extern "C" {
 #define MRF24J40_INTSTAT_TXNIF          (0x01)
 /** @} */
 
+/**
+ * @name    Bitfiled definitions for the GPIO register (0x33)
```

Comment:
please move these two below the INTCON which is `0x32` and before SLPACK which is `0x35`.
Also shouldn't the following defines be `MRF24J40_GPIO_*` to keep the naming consistent?

Reply:
I've done the requested changes.